alloc.cpp

```diff
@@ -429,7 +429,11 @@ void ObjectMemory::FixedSizePool::morePages()
 	// Put the allocation (64k) into the allocation list so we can free it later
```

Comment:
Again, these realloc related changes should not be made - it's going to crash anyway, so no point